projects/ngx-wim/src/lib/wim.service.ts

```diff
@@ -92,7 +92,6 @@ export class WimService {
     // to the class avoids it.
     const msg = message;
     return msg
-      .replace(/&/g, '&amp;')
```

Comment:
I think this is here so that things like `me & you` gets converted to `me &amp; you` to be displayed correctly in HTML. Maybe we can change this to only replace if there are whitespace or line break chars on each side:
```suggestion
      .replace(/([\n\r\s]+)&([\n\r\s]+)/g, '$1&amp;$2')
```

Reply:
I think you're right